examples/mqtt/mqtt-shapes/README.md

```diff
@@ -180,6 +180,12 @@ The following additional external dependencies are required to run the demo:
 
 - [tmux](https://github.com/tmux/tmux) (optional)
 
+Additionally, in macOS® systems you need to install `coreutils`:
+
+    ```sh
+    brew install coreutils
```

Comment:
I forget, but what do we need `coreutils` for? Just wondering if we can refactor the demo to avoid this dependency.

Reply:
I think it was 'expr' and 'shuf', not sure if something else was missing

Reply:
I think `expr` is POSIX so I would expect it to be available, but I can see how `shuf` might be the culprit. Since it's only used in by `shapes_mqtt_publisher.sh` I think it should be possible to refactor the script to use it only if its available (and come up with some alternative if it isn't).

We can leave this for another PR.